### PR TITLE
Docs: Update get-support information

### PR DESCRIPTION
**Pull request by oseasmoran73**
## Description

This PR updates the contents of gets-support page in our dev portal

It also moves it one level up in the navigation bar

https://github.com/department-of-veterans-affairs/VES/issues/562

## Testing

image

image

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]
🚢

File: docs/get-support.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The best way to request support from the Event Bus Team is to reach out to us via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the Digital Service VA workspace.
+
+Please avoid direct messaging an Event Bus team member in Slack. Chatting openly in the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) channel allows others to learn from the questions and answers discussed.
+
+# Slack Channel Support Hours 
+Event Bus Slack channel support hours are from 9am - 5pm ET, Monday through Friday, with the exception of Federal holidays.
+
+- Response times are based on the nature of the posting:
+- Incidents (outage or error): ASAP. Please clearly indicate this is an “INCIDENT” in the post.
```
Comment (madebydna) [inline docs/get-support.md:15]: Perhaps the two items under response types could be listed separately as sub-items, i.e.:

- Response times ...
    - Incidents
    - General inquiries

**Review by madebydna — APPROVED** [review]
Looks good!

File: docs/get-support.md (hunk 1 above)
Comment (oseasmoran73) [reply docs/get-support.md:15]: Oops! Its suppose to be

Response Time
- Incident
- General Inquiries

Thank you for the catch! I also notice the format is a bit funky as I continue to stare.. Will update testing

**Pull request merged**